index.html

```diff
@@ -17,11 +17,18 @@
 <body>
   <header class="header bg">
     <img id="logo" src="images/MyLogo.svg" alt="My logo">
-    <nav class="menu">
+    <nav class="main-nav">
       <img src="images/menu.svg" alt="menu">
+       <ul class="ul-menu">
+         <li>Portoflio</li>
+         <li>About</li>
+         <li>Contact</li>
+       </ul>
     </nav>
   </header>
+
   <section class="intro bg">
+   <div class="wrapper">
     <h1 class="heading">I'm Reem Glad to see you</h1>
     <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and

```

Comment:
- [ ] Being able to replicate a design template into a working page is a skill every front-end developer should have and from your work so far, you are doing a great job. However, please consider making this paragraph and "let's connect" to be spaced correctly so that your design matches that of your chosen template. 

Your design:
<img width="444" alt="chrome_8TTbpcuqf9" src="https://user-images.githubusercontent.com/81468675/144669102-71e750f5-e47c-41ea-a6e2-45c1a8c93f0e.png">

Figma design:
<img width="305" alt="chrome_1KEmnbaqYe" src="https://user-images.githubusercontent.com/81468675/144669157-169d979c-1209-4cb3-9287-fccf75b5f4cd.png">